packages/material-ui/src/Slider/Slider.js

```diff
@@ -714,6 +723,10 @@ const Slider = React.forwardRef(function Slider(props, ref) {
   }, [disabled, handleTouchEnd, handleTouchMove]);
```

Comment:
What about using `stopListening` here too?

Reply:
Yup, absolutely should :+1: